public/Day 225/Game Engine/stack.js

```
@@ -0,0 +1,205 @@
+        /**
+         * THE BACKPROPAGATION ENGINE
+         * Topological sort + recursive gradient accumulation.
+         */
+        backward() {
+            const topo = [];
+            const visited = new Set();
+            
+            const buildTopo = (v) => {
+                if (!visited.has(v)) {
+                    visited.add(v);
+                    v._prev.forEach(child => buildTopo(child));
```

Biome (2.3.14)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)